saleor/product/models.py

```diff
@@ -299,6 +299,12 @@ def sort_by_attribute(
             f"{ordering}name",
         )
 
+    def prefetched_for_webhook(self):
+        return self.prefetch_related(
+            "attributes__values",
+            "attributes__assignment__attribute",
```

Comment:
Are you sure that's all the required fields for `generate_product_payload`?

Reply:
For single object we need only prefetch `attributes`, when we iterating over object then we need to prefetch everything related to that object.